conda/base/context.py

```diff
@@ -314,6 +314,7 @@ class Context(Configuration):
     target_prefix_override = ParameterLoader(PrimitiveParameter(''))
 
     unsatisfiable_hints = ParameterLoader(PrimitiveParameter(True))
+    unsatisfiable_hints_check_depth = ParameterLoader(PrimitiveParameter(3))
```

Comment:
maybe this is more useful if set to 1? That way it will only show the first level of incompatibilities

Reply:
Makes sense to me.  I think that's worth doing, and we'll just have to learn from experience on whether that's what people need.

Reply:
2 is lowest without causing any test breakages. Could change the tests to set this config, or leave the default at 2?